src/Unidevel.OpenWeather/Unidevel.OpenWeather.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>net7.0</TargetFramework>
```

Comment:
Add also <LangVersion>latest</LangVersion>

This should enable nullable support in entire project and cause a lot of warning in modern C#. Analyse where you should use nullable types (one with ? at the end).